design/baremetal-operator/unmanaged-annotation.md

```diff
@@ -0,0 +1,105 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Add support for Unmanaged annotation
+
+## Status
+
+implementable
+
+## Summary
+
+Provide a way to prevent management of BaremetalHost resources
+after provisioning is completed, to facilitate the pivot of
+BaremetalHost resources in a multi-tier deployment.
+
+## Motivation
+
+In a multi-tier deployment where one cluster deploys another, the "parent"
+cluster will contain BMH resources for initial provisioning,
+but the "child" cluster will later contain BMH resources that reference the
+same physical hosts.
+
+In this scenario it's necessary to prevent management operations from the
+parent cluster such as asserting power state, or BMH actions on the child
+cluster such as the [reboot annotation](reboot-interface.md) may fail due
+to unwanted BMC interactions from the parent cluster.
+
+There is an existing
+[pause annotation](https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#pausing-reconciliation)
+which pauses reconciliation of the BMH resources, but this does not remove
+the underlying Ironic host, so power management actions may be performed
+even when the BMH is marked as `paused`.
+
+### Goals
+
+* Add an API to disable management of a BMH on the parent cluster, including
+  all power management actions
+* Make it possible to delete a BMH resource with this "unmanaged" setting, without
+  triggering deprovisioning
+* Ensure it is possible to restart management of BMH resources (in the case
+  where they are not deleted from the parent cluster)
+* Avoid changing behavior of the existing `paused` annotation since
+
+### Non-Goals
+
+* Any coordination between the tiers of BMH resources, that has to be handled externally
+
+## Proposal
+
+### Expected workflow
+
+* User creates BMH resource(s) in parent cluster
+* Provisioning of BMH triggered, which results in a running child cluster
+* Parent cluster BMH resources marked as unmanaged
+* Child cluster BMH resources created, with BMC credentials, but marked
+  externallyProvisioned: true
+
+At this point, the physical hosts are owned by the child cluster BMH, but the
+inventory still exists in the parent cluster.
+
+In the event that all of the child cluster hosts require reprovisioning, it
+would be necessary to remove the maintenance/unmanaged flag, or
+delete/re-create the BMH resource so that the process starts again at the
+beginning.
```

Comment:
This statement could be made clearer. Do these operations "remove the maintenance/unmanaged flag" and "delete/re-create the BMH resource" both happen on the parent cluster when the child cluster is being reprovisioned?

Reply:
Thanks, I clarified this in the latest version.